Week-7/Assignment7.1.js

```diff
@@ -0,0 +1,51 @@
+var head;
+  
+     class Node {
+        constructor(val) {
+            this.data = val;
+            this.next = null;
+        }
+    }
+  
+    /* Function to reverse the linked list */
+    function reverse(node) {
+    var prev = null;
```

Comment:
make use of block scope let and const other places as well

Reply:
yaa, done